webapp/src/components/surveyTimeSelector/surveyTimeSelector.tsx

```diff
@@ -32,14 +33,21 @@ const SurveyTimeSelector = ({value, onChange}: Props) => {
         return timeStrings;
     }, []);
 
+    const [dropdownValue, setDropdownValue] = useState<DropdownOption>(options[18]);
+    useEffect(() => {
+        if (value) {
+            setDropdownValue({value, label: value});
+        }
+    }, [value]);
```

Comment:
Similar comment about my previous one. You can use `useMemo` here instead of using a state.

In this case we don't want to remove the `useMemo` because if not, we would be passing a new object to the prop on every render.

Reply:
Yeah, I agree with this to get rid of the extra render here. I was a bit confused as to why we had value as both a prop and as state when reading this before

Reply:
Done.